### PR TITLE
Fix rsync uploader error and add facebook captcha to global igset

### PR DESCRIPTION
**Pull request by falconkirtaran**

### CODE REVIEW

File: INSTALL.pipeline [hunk 1]
```diff
@@ -14,7 +14,7 @@ Quick install, for Debian and Debian-esque systems:
     sudo apt-get install build-essential python3-dev python3-pip \
       libxml2-dev libxslt-dev zlib1g-dev libssl-dev libsqlite3-dev \
       libffi-dev git tmux fontconfig-config fonts-dejavu-core \
-      libfontconfig1 libjpeg-turbo8 libjpeg8
+      libfontconfig1 libjpeg-turbo8 libjpeg8 lsof
```
Comment (yipdw) [inline INSTALL.pipeline]: What part of the pipeline requires lsof?  Is this a nice-to-have thing?

File: db/ignore_patterns/global.json [hunk 2]
```diff
@@ -122,6 +122,7 @@
         "^https?://(ssl\\.|www\\.)?reddit\\.com/(login\\?dest=|submit\\?|static/button/button)",
         "^https?://digg\\.com/submit\\?",
         "^https?://(www\\.)?facebook\\.com/(plugins/like(box)?\\.php|sharer/sharer\\.php|sharer?\\.php|dialog/(feed|share))\\?",
+        "^https?://www\.facebook\.com/captcha/",
```
Comment (yipdw) [inline db/ignore_patterns/global.json]: This needs to be `\\.`; a single `\.` results in an invalid escape character.

File: INSTALL.pipeline (hunk 1 above)
Comment (yipdw) [inline INSTALL.pipeline]: Oh, I see what needs `lsof` -- didn't look down.

**Comment by falconkirtaran** [comment]
Oops!  That one slipped through I see.  Corrected now.

**Pull request merged**